src/main/java/com/lnu/src/thread/PassengerRandomizer.java

```diff
@@ -9,15 +9,15 @@
 import java.util.List;
 import java.util.concurrent.ThreadLocalRandom;
 
-public class PassengerGenerator extends Thread {
+public class PassengerRandomizer extends Thread {
 
   private static final Logger LOGGER = LogManager.getLogger();
   private final float chance;
   private final int time;
   private final List<Floor> floors;
 
 
-  PassengerGenerator(float chance, int time, List<Floor> floors) {
+  public PassengerRandomizer(float chance, int time, List<Floor> floors) {
```

Comment:
Why Randomizer not Generator?

Reply:
cause we've got such name in our UML diagram